hack/observability/kube-state-metrics/metrics/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Let's add a readme file in this directory with instructions about how to generate the metrics file.
It could also include a disclaimer specifying this is temporary and the long-term goal is to have method to auto-generate metric configuration from API type markers